contracts/src/v0.8/operatorforwarder/test/operator.t.sol

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+import {Operator} from "../Operator.sol";
+import {ChainlinkClientHelper} from "./testhelpers/ChainlinkClientHelper.sol";
+import {LinkToken} from "../../shared/token/ERC677/LinkToken.sol";
+
+import "./testhelpers/Deployer.t.sol";
+import "../AuthorizedReceiver.sol";
+
+contract OperatorTest is Deployer, AuthorizedReceiver {
+  address public s_link;
+  uint256 private dataReceived;
```

Comment:
Nit: private vars also need to start with s_

Reply:
Good catch, thank you